docs/source/exporthumandata.rst

```diff
@@ -44,21 +46,37 @@ like in the Handbook):
 Setting up the OSF Remote
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To use OSF as a storage, you need to provide either your OSF credentials or an OSF access token.
-You can create such a token in your account settings (`Personal access token` and then `Create token`), make sure to create a `full_write` token to be able to create OSF projects and upload data to OSF.
+To use OSF as a storage, you first need to provide either your OSF credentials (username and password) or an OSF access token.
+
+If you choose to use your credentials, proceed as follows:
+
+.. code-block:: bash
+
+    export OSF_USERNAME=YOUR_USERNAME_FOR_OSF.IO
+    export OSF_PASSWORD=YOUR_PASSWORD_FOR_OSF.IO
+
+In this example, we are going to use an OSF access token instead.
+You can create such a token in your account settings (`Personal access token` and then `Create token`).
+Make sure to create a `full_write` token to be able to create OSF projects and upload data to OSF.
 
 .. code-block:: bash
 
     export OSF_TOKEN=YOUR_TOKEN_FROM_OSF.IO
 
-We are now going to use datalad to create a sibling dataset on OSF with name `osf` - this will create a new project called `OSF_PROJECT_NAME` on the OSF account associated with the OSF token in `$OSF_TOKEN`.
+We are now going to use datalad to create a sibling dataset on OSF with name `OSF_PROJECT_NAME`.
+This will create a new project called `OSF_PROJECT_NAME` on the OSF account associated with the OSF token in `$OSF_TOKEN`.
```

Comment:
What is the rational to switch from `osf` to `OSF_PROJECT_NAME` in the example. Do you envision the need to have multiple different OSF projects for the same dataset as a common case?

Having something simple and uniform, such as `osf` makes a lot of sense. Especially in the case of a hierarchy of nested datasets, where one would want to be able to do a `datalad push --to osf --recursive`.

Reply:
> Do you envision the need to have multiple different OSF projects for the same dataset as a common case?

no, I don't think so,

But I would prefer to let users know (also within the example) that it's up to their discretion which name they want to use for their remote and the OSF project. --> after all, it doesn't matter whether it's called `osf` or something else.

But reading this again, it seems like I mixed up some things and it rather should be something like:

> We are now going to use DataLad to create a sibling dataset on OSF as a "special remote". Within git-annex, we will refer to the special remote with the name `$NAME_OF_REMOTE`, while the project that will be created on the OSF account associated with the `$OSF_TOKEN` will be called `$OSF_PROJECT_NAME`.